src/components/banner/banner.jsx

```diff
@@ -0,0 +1,30 @@
+import { React, Fragment } from "react";
+import "./banner.css";
+import bannerImg from "../../Assets/Images/tp29-ning-18-autumn.jpg";
+
+export function Banner() {
+  return (
+    <Fragment>
+      <div>
+        <div className="top-image-container">
+          <img src={bannerImg} alt="Phone-img" className="top-img" />
+          <div className="top-img-overlay rubik-font">
+            Branded SMART PHONES on 40% Discount
+          </div>
+          <div className="brand-intro-container">
+            <h3>
+              <span className="brand-intro-start rubik-font">
+                Smart phones with exciting new features and up to date Specs
+              </span>
+              <span className="brand-intro-end rubik-font">
+                {" "}
+                exclusively available only on
+              </span>
```

Comment:
remove empty braces

Reply:
Needed those braces , React was joining the tags and it made it look like single word